CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/response/UserResponse.kt

```diff
@@ -0,0 +1,60 @@
+package co.nimblehq.coroutine.response
+
+import co.nimblehq.coroutine.model.User
+import com.squareup.moshi.Json
+
+data class UserResponse(
```

Comment:
Why do we have this file and the classes in this module? This should be in the `data` module. 😕

Reply:
@AVI5HEK It should be in `data` module. However, we're aiming to return the original data from `Repository`, that's the reason why there is a bit different compare with the standard.